network/index.js

```diff
@@ -39,17 +52,25 @@ const useNetworkStatus = initialEffectiveConnectionType => {
       const navigatorConnection = navigator.connection;
       const updateECTStatus = () => {
         setNetworkStatus({
-          effectiveConnectionType: navigatorConnection.effectiveType
+          effectiveConnectionType:
+            'onLine' in navigator && navigator.onLine === false
+              ? 'offline'
+              : navigatorConnection.effectiveType,
         });
       };
+
       navigatorConnection.addEventListener('change', updateECTStatus);
+      window.addEventListener('offline', updateECTStatus);
```

Comment:
@JCofman 

Should we have online event listener as well or am I missing something?
For example [rehooks/online-status](https://github.com/rehooks/online-status/blob/master/src/index.js):
```
useEffect(() => {
    window.addEventListener("online", goOnline);
    window.addEventListener("offline", goOffline);

    return () => {
      window.removeEventListener("online", goOnline);
      window.removeEventListener("offline", goOffline);
    };
  }, []);
```